mobilecoind-json/src/bin/main.rs

```diff
@@ -530,28 +530,17 @@ fn submit_tx(
 }
 
 /// Checks the status of a transfer given a key image and tombstone block
-#[post("/tx/status-as-sender", format = "json", data = "<receipt>")]
+#[post("/tx/status-as-sender", format = "json", data = "<submit_response>")]
 fn check_transfer_status(
     state: rocket::State<State>,
-    receipt: Json<JsonSendPaymentResponse>,
```

Comment:
The `JsonSendPaymentResponse` and `JsonSubmitTxResponse` are exactly the same from the Json perspective. The `SubmitTxRespons`e more accurately reflects what will be passed to the mobilecoind service.